config/routes.rb

```diff
@@ -10,6 +10,9 @@
   get "/:id", to: "flow#landing", as: :flow_landing
 
   get "/:id/y/visualise(.:format)", to: "smart_answers#visualise", as: :visualise
+
+  get "/register-a-death/y/overseas/*other", to: redirect("/register-a-death/y/overseas")
```

Comment:
Looks good - one small request, could we please add a comment in here so that it's easy for any future devs (including us) to remember why this is in here? 🙏

Reply:
@FelixMarcusMillne Hey Felix, thanks for the review. We have now added a comment! 👍